src/SetupApi/SetupApi.cs

```diff
@@ -219,6 +219,272 @@ public static partial class SetupApi
+        /// This parameter is optional and can be <see langword="null"/>. If this parameter is specified, <see cref="SetupDiEnumDriverInfo(SafeDeviceInfoSetHandle, SP_DEVINFO_DATA*, DriverType, uint, ref SP_DRVINFO_DATA)"/>
+        /// enumerates a driver list for the specified device. If this parameter is <see langword="null"/>, <see cref="SetupDiEnumDriverInfo(SafeDeviceInfoSetHandle, SP_DEVINFO_DATA*, DriverType, uint, ref SP_DRVINFO_DATA)"/>
+        /// enumerates the global class driver list that is associated with <paramref name="deviceInfoSet"/> (this list is of type
+        /// <c>SPDIT_CLASSDRIVER</c>).
+        /// </param>
+        /// <param name="driverType">
+        /// The type of driver list to enumerate.
+        /// </param>
+        /// <param name="memberIndex">
+        /// The zero-based index of the driver information member to retrieve.
+        /// </param>
+        /// <param name="driverInfoData">
+        /// A caller-initialized <see cref="SP_DRVINFO_DATA"/> structure that receives information about the enumerated driver.
+        /// The caller must set <see cref="SP_DRVINFO_DATA.Size"/> before calling <see cref="SetupDiEnumDriverInfo(SafeDeviceInfoSetHandle, SP_DEVINFO_DATA*, DriverType, uint, ref SP_DRVINFO_DATA)"/>.
+        /// If the <see cref="SP_DRVINFO_DATA.Size"/> member is not properly set, <see cref="SetupDiEnumDriverInfo(SafeDeviceInfoSetHandle, SP_DEVINFO_DATA*, DriverType, uint, ref SP_DRVINFO_DATA)"/> will return
+        /// <see langword="false"/>.
+        /// </param>
+        /// <remarks>
+        /// To enumerate driver information set members, an installer should first call <see cref="SetupDiEnumDriverInfo(SafeDeviceInfoSetHandle, SP_DEVINFO_DATA*, DriverType, uint, ref SP_DRVINFO_DATA)"/> with the
+        /// <paramref name="memberIndex"/> parameter set to <c>0</c>. It should then increment <paramref name="memberIndex"/> and call
+        /// <see cref="SetupDiEnumDriverInfo(SafeDeviceInfoSetHandle, SP_DEVINFO_DATA*, DriverType, uint, ref SP_DRVINFO_DATA)"/> until there are no more values. When there are no more values, the function fails and a call to
+        /// <see cref="Marshal.GetLastWin32Error"/> returns <see cref="Win32ErrorCode.ERROR_NO_MORE_ITEMS"/>.
+        /// </remarks>
+        /// <returns>
+        /// The function returns <see langword="true"/> if it is successful.
+        /// Otherwise, it returns <see langword="false"/> and the logged error can be retrieved with a call to <see cref="Marshal.GetLastWin32Error"/>.
+        /// </returns>
+        /// <seealso href="https://msdn.microsoft.com/en-us/library/windows/hardware/ff551018(v=vs.85).aspx"/>
+        [DllImport(nameof(SetupApi), SetLastError = true, CharSet = CharSet.Unicode)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static unsafe extern bool SetupDiEnumDriverInfo(
+            SafeDeviceInfoSetHandle deviceInfoSet,
+            SP_DEVINFO_DATA* deviceInfoData,
```

Comment:
Add `[Friendly(FriendlyFlags.In | FriendlyFlags.Optional)]` to generate more friendly overloads for this parameter.